app/controllers/api/v3/participants_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
       end
 
       def resume
-        service = ::Participants::Resume.new(participant_action_params)
+        service = ::Participants::Resume.new(participant_action_params.slice(*::Participants::Resume.attribute_names))
```

Comment:
Instead of slicing the attributes for all actions/controllers, would it be cleaner to modify the core `Action` model with a convenience method that we can also easily test? something like:

```
def self.new_filtering_attributes(attributes = {})
  filtered_attributes = attributes.slice(*attribute_names)
  new(filtered_attributes)
end
```